exams/pearson/readers.py

```diff
@@ -54,6 +54,27 @@ def parse_datetime(cls, dt):
         """
         return datetime.strptime(dt, PEARSON_DATETIME_FORMAT).replace(tzinfo=pytz.UTC)
 
+    @classmethod
+    def map_cell(cls, row, source, target, transformer=None):
```

Comment:
Just one question: why is this defined as a `classmethod` if you access it from another method via `self`